src/publishers/abstract_publisher.jl

```diff
@@ -24,10 +28,14 @@ Specifies a publisher along with specific message type.
 This is useful for tracking multiple messages at once
 """
 struct PublishedMessage
-    msg::ProtoBuf.ProtoType
+    msg::Union{ProtoBuf.ProtoType, AbstractVector{UInt8}}
```

Comment:
Might be worth defining this as a custom type, e.g.
```
MercuryMessage = Union{ProtoBuf.ProtoType, AbstractVector{UInt8}}
```

Reply:
Yeah I was thinking the same thing actually. It got kinda tiring to write the union thingy. Ill change that when i re-configure the subscribe stuff